client/post-editor/post-editor.jsx

```diff
@@ -451,7 +492,12 @@ var PostEditor = React.createClass( {
 			content: autosave.content
 		};
 
+		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
 		actions.edit( edits );
+
+		this.props.setTitle( autosave.title );
+		this.props.setExcerpt( autosave.excerpt );
+		this.props.setContent( autosave.content );
```

Comment:
Either way, API call does not start until we cal `save` or `autosave` as I understand.
We are not initiating API calls with any of those, we just set the state of our current memory object, which is local.
This calls are not touching API.
I made them granular so that we can re-use them all over the app (for example title)